packages/client/layouts/image-contain.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
```

Comment:
I think I am ok to have the rest. However, this file feels a bit too specific. Do you have any good use cases?

Reply:
Thanks. For image-contain, I don't have any specific use cases, they were all alternatives. The best practical usage from my point of view would be from setting prop from existing layouts. 

I have dropped that commit & updated the PR.